test/unit_tests/datasets/test_dataset.py

```diff
@@ -0,0 +1,55 @@
+# Authors: Maciej Sliwowski <maciek.sliwowski@gmail.com>
+#
+# License: BSD (3-clause)
+
+import unittest
```

Comment:
@sliwy Could you switch to using pytest? Everywhere else we are using pytest, for setup stuff it has pytest.fixture (https://docs.pytest.org/en/latest/fixture.html). We use it for example here https://github.com/robintibor/braindecode-1/blob/47185b891ca96532095c71d80df9741b3aad77dc/test/unit_tests/datautil/test_windowers.py#L17